app/constants/colors.js

```diff
@@ -2,12 +2,15 @@ const colors = {
   BLACK: '#000',
   WHITE: '#FFF',
   DODGER_BLUE: '#428AF8',
+  mainBlue: '#428AF8',
   SILVER: '#BEBEBE',
   TORCH_RED: '#F8262F',
   MISCHKA: '#E5E4E6',
   PRIMARY_TEXT: '#000',
-  SUCCESS: '#41dca4', // Green
-  WARNING: '#ffc000', // Orange
+  SUCCESS: '#41dca4',
+  green: '#41dca4',
```

Comment:
The naming convention must be homogeneous, either all uppercase or all lowercase